contrib/cmd/runkperf/commands/warmup/command.go

```diff
@@ -76,6 +76,10 @@ var Command = cli.Command{
 			Usage:  "Indicates the target kubernetes cluster is EKS",
 			Hidden: true,
 		},
+		cli.BoolFlag{
```

Comment:
maybe we can update `core-warmup-ready-threshold` option. like if you want to force run it, you can set it to 65536 cores.